docs/content/Reference/Configuration/Config.mdx

```diff
@@ -760,6 +760,89 @@ module.exports = {
 This property controls how often Websocket client subscriptions are refreshed.
 Defaults to `5000`.
 
+### <--{"id" : "Options Reference"}--> semanticLayerSync
+
+This function provides the configuration for [Semantic Layer Sync][ref-sls].
+
+Called once when a refresh worker starts. Can be an `async` function.
+Returns an array with options for specific BI tools:
+
+```typescript
+semanticLayerSync: () =>
+  SemanticLayerSyncConfig[] |
+  Promise<SemanticLayerSyncConfig[]>;
+
+type SemanticLayerSyncConfig =
+  PresetSemanticLayerSyncConfig |
+  SupersetSemanticLayerSyncConfig |
+  MetabaseSemanticLayerSyncConfig;
+
+// Semantic Layer Sync with Preset
+interface PresetSemanticLayerSyncConfig {
+  name: string;
+  active?: boolean;
+  type: "preset";
+  config: {
+    api_token: string;
+    api_secret: string;
+    workspace_url: string;
+  }
+}
+
+// Semantic Layer Sync with Apache Superset
+interface SupersetSemanticLayerSyncConfig {
+  name: string;
+  active?: boolean;
+  type: "superset";
+  config: {
+    api_token: string;
+    api_secret: string;
+    url: string;
+  }
+}
+
+// Semantic Layer Sync with Metabase
+interface MetabaseSemanticLayerSyncConfig {
+  name: string;
+  active?: boolean;
+  type: "metabase";
+  config: {
+    user: string;
+    password: string;
+    url: string;
+  }
+}
+```
+
+Example configuration with an active Preset sync and a disabled Metabase sync:
+
+```javascript
+module.exports = {
+  semanticLayerSync: () => {
```

Comment:
@keydunov With our direction towards using snake case globally, it feels weird to add `semanticLayerSync` instead of `semantic_layer_sync`. This is the last chance to reconsider for this feature. WDYT? I see no harm in introducing some inconsistency now if we're determined to do the switch soon.